src/lib/coaler/embedder/ConformerEmbedder.cpp

```diff
@@ -89,4 +137,118 @@ namespace coaler::embedder {
         params.clearConfs = false;
         return params;
     }
+
+    /*----------------------------------------------------------------------------------------------------------------*/
+
+    std::vector<multialign::PoseID> ConformerEmbedder::generateNewPosesForAssemblyLigand(
+        RDKit::ROMol *worstLigandMol, const multialign::LigandVector &targets,
+        const std::unordered_map<multialign::LigandID, multialign::PoseID> &conformerIDs) {
```

Comment:
Da du hier multialign classes rein gibst, kann das mMn eher in den multialign namespace und dann als eigene klasse a la `assemblyPoseOptimizer`

Reply:
postponed